src/sentry/api/urls.py

```diff
@@ -1888,11 +1887,6 @@
                     AppStoreConnect2FactorAuthEndpoint.as_view(),
                     name="sentry-api-0-project-appstoreconnect-2fa",
                 ),
-                url(
-                    r"^(?P<organization_slug>[^\/]+)/(?P<project_slug>[^\/]+)/appstoreconnect/(?P<credentials_id>[^\/]+)/$",
```

Comment:
I believe is this here

Reply:
I'm still getting the property `encrypted` from this endpoint which causes the symbolicator validation to fail